baselines/fedprox/pyproject.toml

```diff
@@ -141,3 +114,59 @@ exclude = [
 
 [tool.ruff.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "itsmaddox-j"
+
+[tool.flwr.app.components]
+serverapp = "fedprox.server_app:app"
+clientapp = "fedprox.client_app:app"
+
+[tool.flwr.app.config]
```

Comment:
We typically use `a-b-c` as opposed to `a_b_c` for all items in `pyproject.toml`. I was able to make the code work by just ensuring [this line](https://github.com/maddox-j/flower/blob/44b0e6f67e438f748fc6bc6a5dbd91d01d277d8f/baselines/fedprox/fedprox/utils.py#L27) also does `.replace("-", "_")` for the `name` key. All the other `.toml` files will need of a similar modification.